tests/test_ci/TestControllers.py

```diff
@@ -183,25 +183,64 @@ def test_customizedtest_added_to_queue(self):
         self.assertNotIn(1, customized_test)
 
     @mock.patch('mailer.Mailer')
-    def test_inform_mailing_list(self, mock_email):
+    @mock.patch('mod_ci.controllers.get_html_issue_body')
+    def test_inform_mailing_list(self, mock_get_html_issue_body, mock_email):
         """
         Test the inform_mailing_list function
         """
         from mod_ci.controllers import inform_mailing_list
-        from mailer import Mailer
 
+        mock_get_html_issue_body.return_value = """2430 - Some random string\n\n
+                     Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
         email = inform_mailing_list(mock_email, "matejmecka", "2430", "Some random string",
                                     "Lorem Ipsum sit dolor amet...")
 
         mock_email.send_simple_message.assert_called_once_with(
             {
-                'text': '2430 - Some random string\n\n'
-                '        Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n'
-                '        Some random string(https://github.com/Some random string)\n\n\n'
-                '        Lorem Ipsum sit dolor amet...\n        ',
-                'subject': 'GitHub Issue #matejmecka', 'to': 'ccextractor-dev@googlegroups.com'
+                'to': 'ccextractor-dev@googlegroups.com',
+                'subject': 'GitHub Issue #matejmecka',
+                'html': """2430 - Some random string\n\n
+                     Link to Issue: https://www.github.com/test_owner/test_repo/issues/matejmecka\n\n
+                     Some random string(https://github.com/Some random string)\n\n\n
+                     Lorem Ipsum sit dolor amet...\n        """
             }
         )
+        mock_get_html_issue_body.assert_called_once()
+
+    @staticmethod
+    def test_get_html_issue_body():
+        """
+        Test the get_html_issue_body for correct email formatting
+        """
+        from mod_ci.controllers import get_html_issue_body
+
+        title = "[BUG] Test Title"
+        author = "abcxyz"
+        body = ("**strong_text**<br/>*em_text*\n\n"
+                "- [list_with_link](www.example.com)\n"
+                "- [X] checkbox_list with_code\n\n"
+                "`code`\n\n"
+                "> quote")
+        issue_number = 1
+        url = "www.example.com"
+
+        expected_html_issue_body = ("[BUG] Test Title - abcxyz<br/>\n"
```

Comment:
I'd rather see the use of `"""` instead of parentheses

Reply:
The issue with `"""` is that it doesn't account for the identation and hence it looks really bad when we don't want space to be there before the beginning of lines.

Example.
```
def func():
    sentence="""this is first line
this is second line with no space before
                          this is second line with space before but good indentation"""
```
I don't think breaking the indentation will look good.